src/utils/capitalize.js

```diff
@@ -0,0 +1,2 @@
+export const capitalize = str =>
```

Comment:
https://lodash.com/docs/4.17.15#capitalize ne conviendrait pas ?
Sinon le faire en css ?

Reply:
En effet lodash conviendrait, j'ai hésité suite à notre conversation ce matin 😁 
Pas possible en css, ce n'est pas juste pour du style mais bien pour convertir le nom récupéré du fichier JSON, avant d'essayer de l'importer.

Reply:
En rapport à la conversation de ce matin, par contre y'a pas de débat je pense entre ré-inventer la roue et utiliser une lib, go for the lib ;) Si on le fait à la main et qu'on n'utilise pas la lib c'est pour des cas simple. Là tu dois faire une fonction, faire un test avec... on n'est plus dans du simple à mon avis :)

Reply:
On peut supprimer le premier commit du coup non ?